crates/holochain/tests/tests/signals/mod.rs

```diff
@@ -22,6 +22,113 @@ fn to_signal_message(signal: Signal) -> SignalMessage {
     }
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn remote_signals_work_after_sbd_restart() {
+    holochain_trace::test_run();
+
+    const MAX: u64 = 30;
+
+    let vous = SweetLocalRendezvous::new_raw().await;
+
+    let v1: DynSweetRendezvous = vous.clone();
+    let mut c1 =
+        SweetConductor::from_config_rendezvous(SweetConductorConfig::rendezvous(true), v1).await;
+
+    let v2: DynSweetRendezvous = vous.clone();
+    let mut c2 =
+        SweetConductor::from_config_rendezvous(SweetConductorConfig::rendezvous(true), v2).await;
+
+    let dna_file = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::EmitSignal])
+        .await
+        .0;
+
+    let a1 = SweetAgents::one(c1.keystore()).await;
+    let a2 = SweetAgents::one(c2.keystore()).await;
+
+    let (app1,) = c1
+        .setup_app_for_agent("app", a1.clone(), &[dna_file.clone()])
+        .await
+        .unwrap()
+        .into_tuple();
+    let (_app2,) = c2
+        .setup_app_for_agent("app", a2.clone(), &[dna_file])
+        .await
+        .unwrap()
+        .into_tuple();
```

Comment:
I think `SweetConductorBatch::from_rendezvous_config` with a `batch::setup_app` achieves that in fewer lines.

Reply:
No, there's no way to specify the rendezvous instance to use with that method.